include/spirv/unified1/extinst.nonsemantic.clspvreflection.grammar.json

```diff
@@ -0,0 +1,237 @@
+{
+  "revision" : 1,
+  "instructions" : [
+    {
+      "opname" : "Kernel",
+      "opcode" : 1,
+      "operands" : [
+        { "kind" : "IdRef", "name" : "Kernel" },
+        { "kind" : "IdRef", "name" : "Name" }
+      ]
+    },
+    {
+      "opname" : "ArgumentInfo",
+      "opcode" : 2,
+      "operands" : [
+        { "kind" : "IdRef", "name" : "Name" },
+        { "kind" : "IdRef", "name" : "Type Name", "quantifier" : "?" },
```

Comment:
I don't recall seeing multiple optional operands in sequence.
Does the SPIRV-Tools parser handle this? I assume that operands are consumed left to right, so if I only had two operands then it would be TypeName and AddressQualifier.

But if this works, then great.

Reply:
Yes, the parser handles this. Your assumption is correct about multiple optional operands. We already have an instance of this for OpCopyMemory.